### PR TITLE
add: rating feature

### PR DESCRIPTION
**Pull request by ozgurozalp**

### CODE REVIEW

File: app/api/message/route.ts [hunk 1]
```diff
@@ -15,18 +15,14 @@ export async function PUT(req: Request) {
     );
   }
 
-  const { result } = await req.json();
+  const body = await req.json();
```
Comment (zinedkaloc) [inline app/api/message/route.ts:18]: TODO:

- [ ] Later we need to change this, this might effect next features

File: app/layout.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import "./globals.css";
+import "@smastrom/react-rating/style.css";
```
Comment (zinedkaloc) [inline app/layout.tsx:2]: TODO:

- [ ] We might need to update this package.

File: app/profile/projects/page.tsx [hunk 3]
```diff
@@ -1,47 +1,185 @@
...
+                      strokeWidth={2}
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      className="h-4 w-4"
+                    >
+                      <circle cx={12} cy={12} r={10} />
+                      <line x1={2} x2={22} y1={12} y2={12} />
+                      <path d="M12 2a15.3 15.3 0 0 1 4 10 15.3 15.3 0 0 1-4 10 15.3 15.3 0 0 1-4-10 15.3 15.3 0 0 1 4-10z" />
+                    </svg>
+                    <h2 className="whitespace-nowrap text-sm">0 domain</h2>
```
Comment (zinedkaloc) [inline app/profile/projects/page.tsx:151]: Please get it from DB

File: app/profile/projects/page.tsx [hunk 4]
```diff
@@ -1,47 +1,185 @@
...
+                      strokeWidth={2}
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      className="h-4 w-4"
+                    >
+                      <line x1={18} x2={18} y1={20} y2={10} />
+                      <line x1={12} x2={12} y1={20} y2={4} />
+                      <line x1={6} x2={6} y1={20} y2={14} />
+                    </svg>
+                    <h2 className="whitespace-nowrap text-sm">0 clicks</h2>
```
Comment (zinedkaloc) [inline app/profile/projects/page.tsx:170]: Please get it from DB

File: app/profile/projects/page.tsx (hunk 3 above)
Comment (ozgurozalp) [reply app/profile/projects/page.tsx]: done 👍

File: app/profile/projects/page.tsx (hunk 4 above)
Comment (ozgurozalp) [reply app/profile/projects/page.tsx]: done 👍

**Pull request merged**